docs/contributing.rst

```diff
@@ -142,7 +143,7 @@ editing of failing contracts using your preferred text editor. Let's try to brea
     // ----
     // DeclarationError: (36-52): Identifier already declared.
 
-Running ``./test/isoltest`` again will result in a test failure:
+Running ``./test/isoltest`` again results in a test failure:
```

Comment:
I think this should be either `./build/test/tools/isoltest` or `./scripts/isoltest.sh` - please also check the other paths in this file.